vaadin-testbench-core-junit5/src/main/java/com/vaadin/testbench/parallel/ParallelConfigurationStrategy.java

```diff
@@ -0,0 +1,52 @@
+package com.vaadin.testbench.parallel;
+
+import org.junit.platform.engine.ConfigurationParameters;
+import org.junit.platform.engine.support.hierarchical.ParallelExecutionConfiguration;
+import org.junit.platform.engine.support.hierarchical.ParallelExecutionConfigurationStrategy;
+
+import com.vaadin.testbench.Parameters;
+
+/**
+ * Custom configuration strategy using TestBench
+ * {@link Parameters#getTestsInParallel()}.
+ */
+public class ParallelConfigurationStrategy implements
+        ParallelExecutionConfiguration, ParallelExecutionConfigurationStrategy {
+
+    private static final int MAX_CONCURRENT_TESTS;
+
+    static {
+        MAX_CONCURRENT_TESTS = Parameters.getTestsInParallel();
```

Comment:
What is the default value?

Reply:
From `Parameters`:
```
if (hasSystemProperty("testsInParallel")) {
    testsInParallel = getSystemPropertyInt("testsInParallel", 1);
} else if (isLocalWebDriverUsed()) {
    testsInParallel = 10;
} else {
    testsInParallel = 50;
}
```

Reply:
So according to this code, since we don't have "testsInParallel" in default config, the execution would be parallelised into 10 or 50 threads. I did'n see this description in the docs, could you add a sentence there explaining this default behaviour and tell how to override?

Reply:
> Up to 50 test methods are executed simultaneously by default. The limit can be set using the com.vaadin.testbench.Parameters.testsInParallel system property.

Isn't it enough?

Reply:
Yes, and it's worth to tell that if one want to disable concurrent runs without digging into parameters, he can use `@Execution(ExecutionMode.SAME_THREAD)`.

Reply:
Docs updated https://github.com/vaadin/docs/pull/1865/commits/c6751c6b872b2eff1aa5d7ecca0f76162e718035